gcloud/logging/logging_service_v2_api.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/google/googleapis/blob/master/google/logging/v2/logging.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google engineers.
+# Manual additions are allowed because the refresh process performs
+# a 3-way merge in order to preserve those manual additions. In order to not
+# break the refresh process, only certain types of modifications are
+# allowed.
+#
+# Allowed modifications:
+# 1. New methods (these should be added to the end of the class)
+#
+# Happy editing!
+
+from google.api import monitored_resource_pb2
+from google.gax import api_callable
+from google.gax import api_utils
+from google.gax import page_descriptor
+from google.logging.v2 import log_entry_pb2
+from google.logging.v2 import logging_pb2
+
+class LoggingServiceV2Api(object):
+    """Service for ingesting and querying logs."""
+
+    # The default address of the logging service.
+    _SERVICE_ADDRESS = 'logging.googleapis.com'
+
+    # The default port of the logging service.
+    _DEFAULT_SERVICE_PORT = 443
+
+    # The scopes needed to make gRPC calls to all of the methods defined in
+    # this service
+    _ALL_SCOPES = (
```

Comment:
Any reason not to make this public? It'd be nice to have a docstring (Sphinx docs would display it) explaining that these are not all necessary but encompass the entire list that **can** be used.